python/pyarrow/src/arrow/python/arrow_to_pandas.cc

```diff
@@ -2554,6 +2567,14 @@ Status ConvertChunkedArrayToPandas(const PandasOptions& options,
   if (arr->type()->id() == Type::EXTENSION) {
     arr = GetStorageChunkedArray(arr);
   }
+  // In case of a RunEndEncodedArray decode the array
+  else if (arr->type()->id() == Type::RUN_END_ENCODED) {
+    ARROW_ASSIGN_OR_RAISE(arr, GetDecodedChunkedArray(arr));
```

Comment:
We probably should check here for `options.zero_copy_only` and raise an error if set to true (like above is done in case of `strings_to_categorical`)